### PR TITLE
[CI/CD] Update yaml file `./production/helm/loki/values.yaml` (+1 other)

### PR DESCRIPTION
**Pull request by grafanabot**
**Here is a summary of the updates contained in this PR:**
***
Update attribute `$.enterprise.version` in yaml file `./production/helm/loki/values.yaml` to the following value: `v1.7.0`
***
Bump version of Helm Chart
Add changelog entry to `./production/helm/loki/CHANGELOG.md`
Re-generate docs

### CODE REVIEW

**Review by JStickler — APPROVED** [review]
[docs team] LGTM

File: production/helm/loki/values.yaml [hunk 1]
```diff
@@ -1124,10 +1124,12 @@ gateway:
       {{ if .Values.loki.tenants }}
 
 
+
```
Comment (jeschkies) [inline production/helm/loki/values.yaml:1127]: @vlad-diachenko i think this is a bug in your update 😊 it works really well but the new lines seems wrong.

**Comment by MasslessParticle** [comment]
Closing for inactivity

File: production/helm/loki/values.yaml (hunk 1 above)
Comment (jeschkies) [reply production/helm/loki/values.yaml]: It seems to add a new line each time. 

How shall we proceed, @vlad-diachenko

**Pull request closed without merging**